src/style.css

```diff
@@ -1 +1,44 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  padding: 1%;
+}
+
+.logo {
+  margin-left: 2.5%;
+  flex-grow: 3;
+}
+
+.nav__list {
+  display: inline-flex;
+  justify-content: space-evenly;
+  list-style: none;
+}
+
+.nav__item {
+  padding: 20px 10px;
+}
+
+.nav__link {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  color: #000;
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+.nav__link:hover {
+  border-bottom: 2px solid #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
```

Comment:
use `::after ` instead of `border-bottom` and have` position: absolute` and make it position according to `nav__link` (for that you have to give `nav__link` `position: relative`